nemo_rl/data/hf_datasets/dpo.py

```diff
@@ -34,9 +47,15 @@ class DPODataset:
     """
 
     def __init__(self, train_data_path: str, val_data_path: str):
+        warnings.warn(
+            "DPODataset is deprecated and will be removed in a future version. Use PreferenceDataset instead  (see function `to_preference_data_format()` on how to convert your data to this new format).",
```

Comment:
Minor whitespace fix
```suggestion
            "DPODataset is deprecated and will be removed in a future version. Use PreferenceDataset instead (see function `to_preference_data_format()` on how to convert your data to this new format).",
```

Reply:
I updated